docker-compose.yml

```diff
@@ -5,12 +5,14 @@ services:
     build:
       context: .
       dockerfile: docker/Dockerfile
+    container_name: backend
     ports:
       - "8080:8080"
     volumes:
       - ./backend:/root/app
     depends_on:
-      - db
+      db:
+        condition: service_healthy
```

Comment:
Just needed this to ensure the backend doesn't start until the database is fully healthy and listening for connections.